crates/swift-bridge-build/tests/package-test.rs

```diff
@@ -0,0 +1,45 @@
+use std::collections::HashMap;
+use std::fs;
+use std::io::ErrorKind;
+use std::path::Path;
+use std::process::Command;
+
+use swift_bridge_build::{generate_package, GeneratePackageConfig};
+use swift_bridge_build::ApplePlatform as Platform;
+
+#[test]
+fn gen_package() {
+    // Build Rust
```

Comment:
We should delete the `generated` dir at the start of the test so that we're 100% sure that we're starting from a clean state.